client/src/js/addendum/AddendumContainer.jsx

```diff
@@ -80,7 +91,13 @@ function AddendumContainer (props) {
         setAddedIdentities([])
         setRemovedIdentities([])
       })
-      .catch(console.error)
+      .catch(err => {
+        if (err.code === 'permission-denied') {
+          setError('Permission denied, please try again later')
+          return
+        }
+        setError('Request failed, please try again later')
```

Comment:
We should keep track of the error. Before it was printed on the console. If it's not a permission error then we should probably pass that to Bugsnag for proper tracking and debugging